src/lib/config.js

```diff
@@ -6,10 +6,29 @@ import Client from 'mina-signer';
 import { Lightnet, Mina, PrivateKey, PublicKey } from 'o1js';
 import { getBorderCharacters, table } from 'table';
 import Constants from './constants.js';
-import step, { isMinaGraphQlEndpointAvailable } from './helpers.js';
-import prompts from './prompts.js';
+import { readDeployAliasesConfig, step } from './helpers.js';
+import { isMinaGraphQlEndpointAvailable } from './network-helpers.js';
+import { prompts } from './prompts.js';
 
-const log = console.log;
+// Public API
+export default config;
+
+// Private API
```

Comment:
I partially agree with you. It makes no sense to comment so for the modules that have default export (f.e. `config.js`).
On the other hand, we also have helper and other modules without the default export where I'd love to somehow denote / mark as "exist for internal module needs" and "exist for inter-module usage" (APIs that are consumed by other modules).
And then returning to default exports. If we have marker comments in other modules, then it would be fine to leave the same for all of them for consistency.

Another option would be to conditionally export internal methods in case of tests execution:

```js
if (process.env.NODE_ENV === 'test') {
  export { testExports };
}
```

What do you think?

Reply:
cc @ymekuria ☝️

Reply:
I have changed the comments to be `// Module external API` and `// Module internal API (exported for testing purposes)` respectively. 
@ymekuria please let me know if it looks better now.